core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -221,6 +225,16 @@ Timestamp getStart() {
   }
 
   /**
+   * Returns the milliseconds representation of this {@code Duration}.
+   *
+   * @param duration {@code Duration}.
+   * @return the milliseconds representation of a {@code Duration}.
+   */
+  static long toMillis(Duration duration) {
```

Comment:
Currently we don't have such a util class in implcore. It might make sense to put all the static package-private methods together in a util class, though I think that will be in another PR.